jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -485,7 +501,13 @@ def _radial_profile(radial_cutout, reg_bb, centroid, raw=False):
     reg_ogrid = np.ogrid[reg_bb.iymin:reg_bb.iymax, reg_bb.ixmin:reg_bb.ixmax]
     radial_dx = reg_ogrid[1] - centroid[0]
     radial_dy = reg_ogrid[0] - centroid[1]
-    radial_r = np.hypot(radial_dx, radial_dy)[~radial_cutout.mask].ravel()  # pix
+    radial_r = np.hypot(radial_dx, radial_dy)
+
+    # Sometimes the mask is smaller than radial_r
+    if radial_cutout.shape != reg_bb.shape:
+        radial_r = radial_r[:radial_cutout.shape[0], :radial_cutout.shape[1]]

```

Comment:
@larrybradley , in my test case (10 x 10 images dithered by 1 pix with mask off-limit a little for second image when linked by WCS), ApertureStat bounding box shape can be different from the cutout shape. Is this the right way to handle it?